Index: Pages/Index.cshtml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>@page\r\n@model IndexModel\r\n@{\r\n    ViewData[\"Title\"] = \"Home page\";\r\n}\r\n\r\n<div class=\"text-center\">\r\n    <h1 class=\"display-4\">Welkom</h1>\r\n    <a href=\"/Registratie\">test</a>\r\n    <p>Learn about <a href=\"https://docs.microsoft.com/aspnet/core\">building Web apps with ASP.NET Core</a>.</p>\r\n</div>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Pages/Index.cshtml b/Pages/Index.cshtml
--- a/Pages/Index.cshtml	(revision c95afe7a84f69fe07d363d4de2d8ce07c774999c)
+++ b/Pages/Index.cshtml	(date 1621336720824)
@@ -5,7 +5,5 @@
 }
 
 <div class="text-center">
-    <h1 class="display-4">Welkom</h1>
-    <a href="/Registratie">test</a>
-    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
+    <h1 class="display-4">Welkom bij de Digitale Werkplaats Friesland</h1>
 </div>
\ No newline at end of file
Index: Pages/Triage Homepagina.cshtml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Pages/Triage Homepagina.cshtml b/Pages/Triage Homepagina.cshtml
new file mode 100644
--- /dev/null	(date 1621336720856)
+++ b/Pages/Triage Homepagina.cshtml	(date 1621336720856)
@@ -0,0 +1,24 @@
+@page
+@model DWF.Pages.Triage_Homepagina
+
+
+<!DOCTYPE html>
+
+<html>
+<head>
+    <title></title>
+</head>
+<body>
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-6 align-items-center">
+            hoi
+        </div>
+        
+        <div class="col-6">
+            hoi
+        </div>
+    </div>
+</div>
+</body>
+</html>
\ No newline at end of file
